aries-site/src/data/wcag/skeleton.json

```diff
@@ -0,0 +1,513 @@
+    },
+    {
+      "rule": "1.2.8",
+      "label": "Media Alternative (Prerecorded)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.2.9",
+      "label": "Audio-only (Live)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.1",
+      "label": "Info and Relationships",
+      "status": "failed",
+      "notes": "Because skeletons do not have aria-hidden=\"true\" they may mess with the structure of the page for screen readers."
+    },
+    {
+      "rule": "1.3.2",
+      "label": "Meaningful Sequence",
+      "status": "failed",
+      "notes": "Hide skeletons aria-hidden=\"true\" and preserve logical flow."
+    },
+    {
+      "rule": "1.3.3",
+      "label": "Sensory Characteristics",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.4",
+      "label": "Orientation",
+      "status": "passed",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.5",
+      "label": "Identify Input Purpose",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.3.6",
+      "label": "Identify Purpose",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.1",
+      "label": "Use of Color",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.2",
+      "label": "Audio Control",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.3",
+      "label": "Contrast (Minimum)",
+      "status": "not-applicable",
+      "notes": "Technically doesn't apply but recommended to have it be 3:1 contrast with page background"
+    },
+    {
+      "rule": "1.4.4",
+      "label": "Resize Text",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.5",
+      "label": "Images of Text",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.6",
+      "label": "Contrast (Enhanced)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.7",
+      "label": "Low or No Background Audio",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.8",
+      "label": "Visual Presentation",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.9",
+      "label": "Images of Text (No Exception)",
+      "status": "not-applicable",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.10",
+      "label": "Reflow",
+      "status": "passed",
+      "notes": ""
+    },
+    {
+      "rule": "1.4.11",
+      "label": "Non-text Contrast",
+      "status": "failed",
+      "notes": "It depends if skeletons are labeled as graphic objects, if so they must meet 3:1 contrast ratio. This seems like an open ended conversation and there isn't any actual classification for Skeletons. Relevant github discussion (https://github.com/w3c/wcag/issues/2048)."
```

Comment:
can we update the note here to why Skeletons do not need to meet contrast

Reply:
https://medium.com/@accessibilityjillie/day-1-skeleton-loaders-accessibility-ec750fcceba7